tests/integrationtest/r/ddl/default_as_expression.result

```diff
@@ -63,15 +61,23 @@ t2	CREATE TABLE `t2` (
   `c` int(10) DEFAULT NULL,
   `c1` varchar(256) DEFAULT date_format(now(), _utf8mb4'%Y-%m-%d %H.%i.%s')
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
+alter table t0 add column c2 date default (date_format(now(),'%Y-%m'));
+Error 1292 (22007): Incorrect datetime value: '2024-04'
```

Comment:
It's 2024-05 now, the test will fail 😂

Reply:
We handled it in https://github.com/pingcap/tidb/pull/52829/files#diff-0d327ff4958d90dcfabf3217c1813aaa93e0cf9587e3cf51a2eb1b1b090d938aR43.